website/client/components/appFooter.vue

```diff
@@ -71,9 +70,10 @@
         .row
           .col-10 {{ $t('donateText3') }}
           .col-2
-            button.btn.btn-donate(@click="donate()")
-              .svg-icon.heart(v-html="icons.heart")
-              .text {{ $t('companyDonate') }}
+            .btn.btn-contribute
```

Comment:
@rcmbal I think the issue requested only to change the button in /static pages, while I think this PR changes it in all the pages since `appFooter.vue` is used across the entire website

Reply:
Actually Lemoness said that changing it in both places is fine for simplicity. :) https://github.com/HabitRPG/habitica/issues/9399#issuecomment-343346929

Reply:
ah I missed that!